client/components/LearnerViewDirective.js

```diff
@@ -1025,12 +1038,21 @@ tie.directive('learnerView', [function() {
          * @param {string} code
          * @param {string} lang
          */
-        var storeCodeAndUpdateCachedCode = function(
-          questionId, code, lang) {
-          CodeStorageService.storeCode(questionId, code, lang);
+        var storeCodeAndUpdateCachedCode = function(questionId, code, lang) {
+          LocalStorageService.storeCode(questionId, code, lang);
           cachedCode = code;
         };
 
+        var storeFeedback = function() {
+          var latestFeedback =
+            $scope.feedbackStorage[$scope.feedbackStorage.length - 1];
+          LocalStorageService.storeFeedback(
```

Comment:
I feel like it should be okay not to denote that it's the latest feedback in storage service since we aren't planning on storing any other kind of feedback? And to be fair, it's also storing reinforcement bullets as well so I though feedback was just a general catch for having both the latest feedback + reinforcement bullets. Without having to name it something atrocious like storeLatestFeedbackAndReinforcementBullets.